### PR TITLE
modified the testcases to handle fiji array.Fiji api's throws different exception

### PR DESCRIPTION
**Pull request by ar-india**
Signed-off-by: ar-india <alokranjan2005@gmail.com>

### CODE REVIEW

File: tests/nimbleclientbase.py [hunk 1]
```diff
@@ -49,6 +51,29 @@
 os_client = None
 
 
+def is_array_version_above_or_equal(arr_version_to_check):
```
Comment (ar-india) [inline tests/nimbleclientbase.py:54]: have added a logic wherein we now check the array version against which the testcases are running.. there are few testcase which are not supported on lower version of nimOS. hence , this is needed

**Comment by ar-india** [comment]
have added a logic wherein we now check the array version against which the testcases are running.. there are few testcase which are not supported on lower version of nimOS. hence , this is needed

**Review by suneeth51 — APPROVED** [review]
lgtm..

**Review by suneeth51 — APPROVED** [review]
lgtm..

**Pull request merged**